jekyll/_cci2/configuration-reference.md

```diff
@@ -804,7 +804,12 @@ Special step for deploying artifacts.
 
 `deploy` uses the same configuration map and semantics as [`run`](#run) step. Jobs may have more than one `deploy` step.
 
-In general `deploy` step behaves just like `run` with one exception - in a job with `parallelism`, the `deploy` step will only be executed by node #0 and only if all nodes succeed. Nodes other than #0 will skip this step.
+In general `deploy` step behaves just like `run` with two exceptions:
+
+- In a job with `parallelism`, the `deploy` step will only be executed by node #0 and only if all nodes succeed. Nodes other than #0 will skip this step.
+- In a job that runs with SSH, the `deploy` step will not execute, and the following action will show instead:
+  > **skipping deploy**  
```

Comment:
📝 This line has two trailing spaces for a forced linebreak. (Not sure if this will cause issues later.)